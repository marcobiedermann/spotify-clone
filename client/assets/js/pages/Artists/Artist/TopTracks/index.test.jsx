import React from 'react';
import renderer from 'react-test-renderer';
import { TopTracksPage } from '.';

it('renders correctly', () => {
  const tree = renderer
    .create(<TopTracksPage />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
