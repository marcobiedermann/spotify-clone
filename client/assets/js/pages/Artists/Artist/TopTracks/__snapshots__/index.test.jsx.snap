// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <table>
    <tbody />
  </table>
</div>
`;
